runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -148,6 +155,9 @@
 	@Getter(AccessLevel.PACKAGE)
 	private LootTrackerClient lootTrackerClient;
 
+	private Boolean looted = false;
```

Comment:
Why is this a boxed type? Also, `boolean` fields initialize to `false`, so it does not need to be declared.

Reply:
it is Boolean so it can be saved/loaded from the config

Reply:
Mentioned in Discord, but this becomes boxed when being coerced to `Object` by `setConfiguration()`. You can change this to `boolean` and test locally to see for yourself that this works fine that way.